src/runtime-rs/crates/hypervisor/src/utils.rs

```diff
@@ -47,3 +50,15 @@ pub fn get_jailer_root(sid: &str) -> String {
 
     [&sandbox_path, JAILER_ROOT].join("/")
 }
+
+pub fn do_add_netns(netns: &Option<String>) -> Result<()> {
```

Comment:
I propose calling this function just `enter_netns()` since that's precisely what it does.